src/panel/direction/DirectionPanel.jsx

```diff
@@ -180,6 +182,31 @@ export default class DirectionPanel extends React.Component {
     this.setState({ [which]: value, isDirty: true }, this.update);
   }
 
+  setDirectionPoint = poi => {
+
+    // Special case: if both origin and destination are already set, open PoI panel
```

Comment:
Are we sure of that? I know it's in the ticket description, but it seems a bit brutal to me. Currently we just do nothing.